R/calcHDDCDD.R

```diff
@@ -0,0 +1,99 @@
+#' HDD and CDD scenario and historic data
+#'
+#' smoothes the HDD and CDD scenario data
+#'
+#' @param tlimit Temperature threshold for computing HDD and CDD: 18, 21 or 25
```

Comment:
What happens if tlimit is not 18, 21, 25? If NULL/empty magpie is returned maybe we should add `stopifnot(tlimit %in% c(18, 21, 25))`?

Reply:
The comment line is outdated. All temperatures between 17 and 25°C are available. I corrected this and now check the set point temperature.